release/dist/unixpkgs/pkgs.go

```diff
@@ -65,7 +69,9 @@ func (t *tgzTarget) Build(b *dist.Build) ([]string, error) {
 		return nil, err
 	}
 	defer f.Close()
-	gw := gzip.NewWriter(f)
+	h := sha512.New()
+	hw := io.MultiWriter(f, h)
+	gw := gzip.NewWriter(hw)
```

Comment:
As constructed, we'll be hashing the bytes after tar'ing and gzip'ing, right? It looks like that to me, and that's what we want, but it took me a minute to figure that out. It might be good for future readers to put a comment here saying that we intentionally hash the result of gzip (and not, say, the file contents that go into tar or gzip).

Reply:
Good point, left a comment :+1: